docs/kubeedge.md

```diff
@@ -26,10 +26,10 @@ The advantages of Kubeedge include mainly:
 KubeEdge is composed of these components:
 
 - **Edged:** Edged is an agent running on edge node for managing user's application.
-- **[EdgeHub](modules/edgehub):** EdgeHub is a web socket client, which is responsible for interacting with **Huawei Cloud IEF service**, including sync cloud side resources update, report edged side host and device status changes.
-- **[EventBus](modules/eventbus):** EventBus is a MQTT client to interact with MQTT server(mosquitto), offer subscribe and publish capability to other components.
-- **[DeviceTwin](modules/devicetwin):** DeviceTwin is responsible for storing device status and syncing device status to the cloud. It also provides query interfaces for applications.
-- **[MetaManager](modules/metamanager):** MetaManager is the message processor and between edged and edgehub. It's also responsible for storing/retrieving metadata to/from a lightweight database(SQLite). 
+- **[EdgeHub](modules/edgehub.html):** EdgeHub is a web socket client, which is responsible for interacting with **Huawei Cloud IEF service**, including sync cloud side resources update, report edged side host and device status changes.
+- **[EventBus](modules/eventbus.html):** EventBus is a MQTT client to interact with MQTT server(mosquitto), offer subscribe and publish capability to other components.
+- **[DeviceTwin](modules/devicetwin.html):** DeviceTwin is responsible for storing device status and syncing device status to the cloud. It also provides query interfaces for applications.
+- **[MetaManager](modules/metamanager.html):** MetaManager is the message processor and between edged and edgehub. It's also responsible for storing/retrieving metadata to/from a lightweight database(SQLite). 

```

Comment:
MetaManager is the message processor ~~and~~ between edged and edgehub

Reply:
This is fixed.